docs/dev/azure/azure_client_certs_auth.md

```diff
@@ -0,0 +1,37 @@
+# Azure Authentication using Client certificates
+
+Azure accepts client certificates as a means of authentication in the service principal and terraform accepts it too. 
+The installer now supports authentication via the certs after [1] merged. 
+
+### Pitfalls
+Although the installer can now use the certs to authenticate, CCO does not support this and hence the installer
+should create the cluster in manual credentials mode only.
+
+### Prerequisites
+- A certificate that is suitable for Azure is created.
+- Register the certificate with Azure as part of App Registrations in the Azure AD. (Refer [2] for more details)
+
+## Steps
+The installer takes the service principal for authentication and the current requirement is that these fields are
+populated.
+
+- Tenant ID
+- Client ID
+- Client Secret
+- Subscription ID
```

Comment:
I don't think we need to include this. The list of what's required for the cert-based SP should suffice.